internal/resources/clustergroup/cluster_group_full_name.go

```diff
@@ -14,7 +14,7 @@ import (
 
 var ClusterGroupFullname = &schema.Schema{
 	Type:        schema.TypeList,
-	Description: "The schema for cluster group iam policy full name",
+	Description: "The schema for cluster group full name",
```

Comment:
Why are we changing this? I believe the IAM policy full name for a scope was different from the full name of the scope itself. Has there been some refactoring? @ishangupta-ds can you help me understand this change.

Reply:
I think this sub-resource should have a generic description since we are trying to reuse this across resources.

If not then @Shubbhang351 please make this a parameterised function - passing the `withDescription` option as per need.

Reply:
yes we are trying to reuse this across resources